src/singletons/Updates.cpp

```diff
@@ -81,17 +81,19 @@ void Updates::installUpdates()
         assert(false);
         return;
     }
+    auto windowName =
+        QString("%1 - Update").arg(Version::instance().fullVersion());
```

Comment:
is this a good chance to call it `Updater`

Reply:
I didn't change it because that's how it was called. You are free to apply a change to this (or another) branch if you feel it's necessary Okayeg